### PR TITLE
Cleanups for removed classes

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,8 +72,8 @@ At runtime the host launches the plug-in's Activity, the plug-in's Activity retu
                         android:label="@string/plugin_name"
                         android:targetActivity=".ui.activity.PluginActivityImpl">
                     <intent-filter>
-                        <!-- For a plug-in setting, use EDIT_SETTING instead. -->
-                        <action android:name="com.twofortyfouram.locale.intent.action.EDIT_CONDITION"/>
+                        <!-- For a plug-in setting, use ACTION_EDIT_SETTING instead. -->
+                        <action android:name="com.twofortyfouram.locale.intent.ACTION_EDIT_CONDITION"/>
```
Comment (ccjernigan) [inline README.md]: This line introduces a regression.

The field name is "ACTION_EDIT_CONDITION" but the Intent string is "com.twofortyfouram.locale.intent.action.EDIT_CONDITION"

https://github.com/twofortyfouram/android-plugin-api-for-locale/blob/master/pluginApiLib/src/main/java/com/twofortyfouram/locale/api/Intent.java

Comment (sschuberth) [inline README.md]: Oh, that's confusing. But you're right, so I'll drop the commit and force-push.

**Comment by sschuberth** [comment]
@ccjernigan I've updated this PR according to your comment.

**Comment by ccjernigan** [comment]
I've squashed these down into a single commit and merged to our master branch.  Thank you for submitting these fixes!

**Pull request closed without merging**